README.md

```diff
@@ -44,6 +44,21 @@ When launching the app, you can specify the port to use with a command like the
 ```
 ./gradlew bootRun -Pport=8888
 ```
+### Configuring application-related properties
+1. When running the application from sources:
+   
+    Change the [application.properties](https://github.com/parasoft/parasoft-demo-app/blob/main/application.properties) file directly or put a customized version of the file to overwrite the properties and put it under config folder before running the application.
```

Comment:
The link will work after this branch is pused to master.

Reply:
For this paragraph, I see you copied over my comment. I didn't mean to have you copy over, but just give you a layout. Please review the description and give necessary information. For example, put the command line example back.

Reply:
Also please use ">" for notes if that makes it look better.